modular_zubbers/code/game/traits/positive.dm

```diff
@@ -0,0 +1,11 @@
+/datum/quirk/featherweight
+	name = "Featherweight"
+	desc = "Your body is lighter than others'. You can be picked up by the scruff or collar with ease."
+	value = 1
```

Comment:
why does this have a value at all

Reply:
I originally thought it was a _slight_ upgrade to the fireman's carry for someone else, but afterwards, I realized it was just the same.

Kinda didn't update it